_gsocproposals/2019/proposal_sixtrackcoll

```diff
@@ -0,0 +1,30 @@
+---
+title: Hardware acceleration of scattering routines in SixTrack
+layout: gsoc_proposal
+project: SixTrack
+year: 2019
+organization: CERN
+---
+
+## Project Description
+
+When high energy protons in the LHC are intercepted by the collimator jaws, they undergo scattering due to the interactions of the incoming protons with either the electromagnetic field of the atoms, or the strong potential of the nucleus. Presently, a physics model for the scattering mechanisms is implemented in SixTrack using a Monte-Carlo code. The interactions fall into four categories: nuclear interaction, coulomb scattering, ionization and hard electromagnetic processes (e.g. pair production).
+
+
+## Expected results
+Demonstrate that the C code running on a GPU can run the scattering routines at the same speed or faster than on CPU, while keeping bit-level identical results.
+
+## Mentors
+
+  * [Johann A. Briffa](mailto:johann.briffa@um.edu.mt)
```

Comment:
Could you please add an organization description for The University of Malta ?

Reply:
I would use Johann CERN credentials (he is registered under EP-UAI) if ok for you.

Reply:
That's fine, it is up to the mentors to decide about the affiliation they want to have.